backend/src/impl/general_configs/dataset_custom_features.json

```diff
@@ -0,0 +1,9 @@
+{
+  "sst2": {
```

Comment:
should we use `dataset name` or `dataset_id` as the key?

Reply:
Good point, I'll fix this in a future PR when it becomes necessary.